pkg/hostagent/port.go

```diff
@@ -40,18 +40,24 @@ func hostAddress(rule limayaml.PortForward, guest api.IPPort) string {
 	return host.String()
 }
 
-func (pf *portForwarder) forwardingAddresses(guest api.IPPort) (string, string) {
+func (pf *portForwarder) forwardingAddresses(guest api.IPPort) (hostAddr string, guestAddr string) {
 	for _, rule := range pf.rules {
 		if rule.GuestSocket != "" {
 			continue
 		}
 		if guest.Port < rule.GuestPortRange[0] || guest.Port > rule.GuestPortRange[1] {
 			continue
 		}
+
 		switch {
-		case guest.IP.IsUnspecified():
```

Comment:
Why has this case been deleted? Any port bound to `0.0.0.0` (or `::`) should match *any* rule.

Or at least `0.0.0.0` needs to match any rule where the GuestIP is an IPv4 address.

But if we differentiate between `0.0.0.0` and `::` here, then we would create another breaking change.

Reply:
Dan, I'll mark those two that are not marked as outdated so you can review again, aight?